Classes/Service/Notification/NotificationService.php

```diff
@@ -97,6 +98,7 @@ public function notifySubscribers(SubscribeableInterface $subscriptionObject, No
 		}
 
 		$topicLink = $this->uriBuilder->setTargetPageUid($this->settings['pids']['Forum'])->setArguments($arguments)->build();
+        $topicLink = \TYPO3\CMS\Core\Utility\GeneralUtility::locationHeaderUrl($topicLink);
```

Comment:
:+1: on using absolute URIs. But since we are already using the `UriBuilder`, why not just add `->setCreateAbsoluteUri(true)` to the call chain above?

```php
$topicLink = $this->uriBuilder
    ->setTargetPageUid($this->settings['pids']['Forum'])
    ->setArguments($arguments)
    ->setCreateAbsoluteUri(true)
    ->build();
```